<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketsAppClass</class>
 <widget class="QMainWindow" name="SocketsAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SocketsApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="input1">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>0</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="ok_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="outputField">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHost_to_IP"/>
   <addaction name="actionIP_to_Host"/>
   <addaction name="actionService_to_Port"/>
   <addaction name="actionPort_to_Service"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHost_to_IP">
   <property name="text">
    <string>Host to IP</string>
   </property>
   <property name="toolTip">
    <string>Translate a host name to an IP address</string>
   </property>
  </action>
  <action name="actionIP_to_Host">
   <property name="text">
    <string>IP to Host</string>
   </property>
   <property name="toolTip">
    <string>Translate an IP to a host name</string>
   </property>
  </action>
  <action name="actionService_to_Port">
   <property name="text">
    <string>Service to Port</string>
   </property>
   <property name="toolTip">
    <string>Translate a service/protocal to a port number</string>
   </property>
  </action>
  <action name="actionPort_to_Service">
   <property name="text">
    <string>Port to Service</string>
   </property>
   <property name="toolTip">
    <string>Translate a port or protocol to a service</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SocketsApp.qrc"/>
 </resources>
 <connections/>
</ui>
